<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_tap_payment</class>
 <widget class="QWidget" name="page_tap_payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_previous_page">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_to_idle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>1760</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="order_total_amount">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>1105</y>
     <width>250</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
background-image: url(background.png);
font-style: light;
font-weight: bold;
font-size: 36px;
line-height: 44px;

color: #58595B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="order_drink_amount">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>1470</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
font-style: light;
font-size: 24px;
color: #58595B;</string>
   </property>
   <property name="text">
    <string>order_drink_amount</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_payment_bypass">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>1730</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1470</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
font-style: light;
font-size: 24px;
color: #58595B;</string>
   </property>
   <property name="text">
    <string>order summary</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>700</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="animated_Label">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>327</y>
     <width>639</width>
     <height>639</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="preauthLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>1090</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
font-style: light;
font-size: 24px;
color: #58595B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
